main.py

```diff
@@ -29,7 +31,8 @@
     st.title("Lux AI submission statistics")
     col1, col2 = st.columns(2)
 
-    sub_id = col1.number_input("Submission ID", value=30131764, step=1, key="sub_id")
+    sub_id = col1.number_input("Submission ID", value=int(query_params['id'][0]), step=1, key="sub_id")

```

Comment:
An empty url query leads to a crash now. Consider fallback to default submission id
```
DEFAULT_SUB_ID = 30512132
sub_id = col1.number_input("Submission ID", value=int(query_params.get("id", [DEFAULT_SUB_ID])[0]), step=1, key="sub_id")
```